intbot/core/tasks.py

```diff
@@ -70,3 +71,9 @@ def process_github_webhook(wh: Webhook):
     )
     wh.processed_at = timezone.now()
     wh.save()
+
+
+def process_zammad_webhook(wh: Webhook):
+    # NOTE(artcz) Do nothing for now. Just a placeholder.
+    # Processing will come in the next PR.
+    return None
```

Comment:
[nitpick] I think this is the case to use `return` instead of `return None` - both are functionality-wise the same, but my feeling was that `return None` indicates that `None` is later used, while `return ` is used like `break` in loops. I might be wrong and you might have a different preference - if so, feel free to disregard.